vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/treegrid/demo/TreeGridDemo.java

```diff
@@ -148,4 +151,22 @@ public boolean hasChildren(Account item) {
 
         addCard("TreeGrid with lazy loading", grid);
     }
+
+    private void createTreeGridWithComponentsInHierarchyColumnUsage() {
```

Comment:
This example is a bit confusing to me, because the links seem to work on leaf nodes. Also, I think it would be nice to show more hierarchy levels.

Reply:
I think it would be better to have more data on the Grid. For example, you can add "Plain Java", "CDI" and "Spring" to the "New project" and also you can use two or three components in the "UI Components".

Reply:
Added more levels.
@tulioag the fact that clicking on the link in parent row is related with this issue: https://github.com/vaadin/vaadin-grid/issues/1582

Reply:
I understand, but since this is the current behavior of grid, I think it makes the demo look broken. There is a visual indication in all items that it is a link, but it only works as such on the leaf nodes.

Reply:
I changed the demo example